static/app/components/forms/selectOption.tsx

```diff
@@ -12,47 +13,52 @@ function SelectOption(props: Props) {
   const {label, data, selectProps, isMulti, isSelected, isFocused, isDisabled} = props;
   const {showDividers, verticallyCenterCheckWrap} = selectProps;
   const {
+    value,
     details,
+    tooltip,
+    tooltipDelay,
     leadingItems,
     trailingItems,
     leadingItemsSpanFullHeight,
     trailingItemsSpanFullHeight,
   } = data;
 
   return (
-    <selectComponents.Option {...props} className="select-option">
-      <InnerWrap isFocused={isFocused} isDisabled={isDisabled}>
-        <Indent isMulti={isMulti} centerCheckWrap={verticallyCenterCheckWrap}>
-          <CheckWrap isMulti={isMulti} isSelected={isSelected}>
-            {isSelected && (
-              <IconCheckmark
-                size={isMulti ? 'xs' : 'sm'}
-                color={isMulti ? 'white' : undefined}
-              />
+    <selectComponents.Option className="select-option" {...props}>
+      <Tooltip containerDisplayMode="block" title={tooltip} delay={tooltipDelay ?? 500}>
```

Comment:
adding an optional tooltip, for cases like this

<img width="250" alt="Screen Shot 2022-04-26 at 12 17 56 PM" src="https://user-images.githubusercontent.com/44172267/165375723-385a045d-f829-4470-a67a-f11b8c2a21c6.png">